### DISCUSSION TITLE
blog/daily/[MyLog]%20-%20%EC%B2%AB%EC%A4%84

### DISCUSSION THREAD

**Discussion by giscus** [swj9707/swj_techblog#2] [Reply]
# blog/daily/[MyLog]%20-%20%EC%B2%AB%EC%A4%84

Next.js, Tailwind css 기반 블로그를 시작하며

https://swj-techblog.vercel.app/blog/daily/[MyLog]%20-%20%EC%B2%AB%EC%A4%84

**Comment by oxspan** [comment]
# 댓글테스트
마크다운으로도 가능하지요!
